### ISSUE TITLE
Resource Binding

### ISSUE THREAD

**Issue by AndersGM** [shred/acme4j#80]
According to the documentation ([https://shredzone.org/maven/acme4j/usage/login.html](https://shredzone.org/maven/acme4j/usage/login.html)) it is possible to bind to a resource using login - e.g. 
```java
Authorization` auth = login.bindAuthorization(authorizationURL);
```
The documentation states, that one can do so for Authorization, Certificate, Order, and Challenge.
However, I don't see a bindChallenge.
I can usee createChallenge, with a JSON structure, but thats not as smooth as the bindChallenge approach.
Thanks for a great piece of work!

**Comment by shred** [comment]
You are right! A `bindChallenge()` is supposed to be there, but actually it isn't. :confused: I will have a look into it.

At the moment, the best way is to bind the `Authorization` and get the `Challenge` from there. But that's not how it was meant to be.

**Comment by AndersGM** [comment]
@shred Thanks for your response :) I keep an internal state of the challenges, so the other approach would be preferable. Will check if I can create a PR myself

**Comment by shred** [comment]
Closed via your PR #83. Thank you!

**Comment by shred** [comment]
I have just released V2.8 with your change. Thank you for your patience! :wink:

**Comment by AndersGM** [comment]
Thank you very much :)
